### DISCUSSION TITLE
Support for two organizations in thehive

### DISCUSSION THREAD

**Discussion by idolaman** [Security-Onion-Solutions/securityonion#3404] [Unsupported Versions]
Does SO Support for two organizations in thehive?
We have 2 seperate teams and we want to know if this feature is possible or will be possible in future.

Thanks!

**Comment by presianbg** [comment]
I was thinking about the same, but by integrating with external TheHive v4 server? 
Do you know if that's an option ?

**Reply by weslambert** [reply]
This is not supported OOB at the moment.

**Comment by weslambert** [comment]
You can add another organization if you like.  We only create the `securityonion` organization.
